modules/log6x-6-2-0-rn.adoc

```diff
@@ -6,52 +6,62 @@
 
-* With this update, the default memory limit for the collector increases from 1024 Mi to 2048 Mi. Users should adjust resource limits based on their cluster’s specific needs and specifications. (link:https://issues.redhat.com/browse/LOG-6072[LOG-6072])
+=== Log Collection
 
-* With this update, users can now set the syslog output delivery mode of the `ClusterLogForwarder` CR to either `AtLeastOnce` or `AtMostOnce.` (link:https://issues.redhat.com/browse/LOG-6355[LOG-6355])
+* With this update, HTTP outputs include a `proxy` field that you can use to send log data via an HTTP proxy. (link:https://issues.redhat.com/browse/LOG-6069[LOG-6069])
 
 === Log Storage
 
-* With this update, the new `1x.pico` LokiStack size supports clusters with fewer workloads and lower log volumes (up to 50GB/day). (link:https://issues.redhat.com/browse/LOG-5939[LOG-5939])
+* With this update, time-based stream sharding in Loki is now enabled by the {loki-op}. This solves the issue of ingesting log entries older than the sliding time-window used by Loki. (link:https://issues.redhat.com/browse/LOG-6757[LOG-6757])
 
-////
+* With this update, you can configure a custom certificate authority (CA) certificate with {loki-op} when using Swift as an object store. (link:https://issues.redhat.com/browse/LOG-4818[LOG-4818])
+
+* With this update, you can configure workload identity federation on {gcp-first} by using the Cluster Credential Operator in OpenShift 4.17 and later releases with the {loki-op}. (link:https://issues.redhat.com/browse/LOG-6158[LOG-6158])
 
 [id="logging-release-notes-6-2-0-technology-preview-features_{context}"]
 == Technology Preview
 
 :FeatureName: The OpenTelemetry Protocol (OTLP) output log forwarder
 include::snippets/technology-preview.adoc[]
 
-////
-* With this update, a `dataModel` field has been added to the `lokiStack` output specification. Set the `dataModel` to `Otel` to configure log forwarding using the OpenTelemetry data format. The default is set to `Viaq`. For information about data mapping see link:https://opentelemetry.io/docs/specs/otlp/[OTLP Specification].
+* With this update, OpenTelemetry support offered by OpenShift {logging-uc} continues to improve, specifically in the area of enabling migrations from the ViaQ data model to OpenTelemetry when forwarding to LokiStack. (link:https://issues.redhat.com/browse/LOG-6146[LOG-6146])
 
-////
+* With this update, the field `structuredMetadata` has been removed from {loki-op} in the `otlp` configuration because structured metadata is now the default type. Additionally, the update introduces a `drop` field that administrators can use to drop OpenTelemetry attributes when receiving data through OpenTelemetry protocol (OTLP). (link:https://issues.redhat.com/browse/LOG-6507[LOG-6507])
```

Comment:
```suggestion
* With this update, the `structuredMetadata` field has been removed from {loki-op} in the `otlp` configuration because structured metadata is now the default type. Additionally, the update introduces a `drop` field that administrators can use to drop `OpenTelemetry` attributes when receiving data through `OpenTelemetry` protocol (OTLP). (link:https://issues.redhat.com/browse/LOG-6507[LOG-6507])
```